eosiojava/src/main/java/one/block/eosiojava/session/TransactionProcessor.java

```diff
@@ -887,6 +908,43 @@ private AbiEosSerializationObject serializeAction(Action action, String chainId,
         return actionAbiEosSerializationObject;
     }
 
+    private AbiEosSerializationObject deserializeActionTraceReturnValue(ActionTrace actionTrace, String chainId, IABIProvider abiProvider)
+            throws DeserializeReturnValueError {
+        String actionAbiJSON;
+        try {
+            actionAbiJSON = abiProvider
+                    .getAbi(chainId, new EOSIOName(actionTrace.getAccountName()));
+        } catch (GetAbiError getAbiError) {
+            throw new DeserializeReturnValueError(
+                    String.format(ErrorConstants.TRANSACTION_PROCESSOR_GET_ABI_ERROR,
+                            actionTrace.getAccountName()), getAbiError);
+        }
+
+        Abi abi = Utils.getGson(DateFormatter.BACKEND_DATE_PATTERN).fromJson(actionAbiJSON, Abi.class);
+        String returnType = abi.getActionReturnTypeByActionName(actionTrace.getActionName());
+
+        AbiEosSerializationObject actionAbiEosSerializationObject = new AbiEosSerializationObject(
+                actionTrace.getAccountName(), actionTrace.getActionName(),
```

Comment:
Do we want to consider constructor overloading here for AbiEosSerializationObject since the account name and action name are not required?

Reply:
Yeah, I can add that. My initial concern was that there was some hidden business logic that didn't allow for null account/action names. I just ran a test on the abieos side and it looks like it works. I'll also update https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/pull/51 with explicit tests for this. The current tests don't pass in null for those fields.

Reply:
New tests: https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/pull/51/files#diff-640ac0a6426edefb132dbf8b87d3b6beR188 + https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/pull/51/files#diff-640ac0a6426edefb132dbf8b87d3b6beR294

Reply:
@GonnaGitYou Updated in `15bb0b3` and `6743977`. Was going to make the main constructor private, but that ends up being a breaking change.